content/docs/product/docs/_index.md

```diff
@@ -0,0 +1,176 @@
+---
+title: How to write documentation
+linkTitle: Documentation
+weight: 5
+description: >
+  This section explains how to document functions of our product for our end users.
+---
+
+In this section you learn how to add to and maintain the documentation for our product. We assume here that you are part of a product team or a SIG and want to maintain the content your team/SIG is responsible for.
+
+## Public by default
+
+When we talk about documentation here, we talk about everything under our public documentation site, which is [docs.giantswarm.io](https://docs.giantswarm.io/).
+
+Our goal is to have all documentation for our product be available via that site. This is towards the mission that every question should be answerable by a link to the appropriate document.
+
+## Responsibilities
+
+Our responsibilities for public documentation are layered.
+
+- Ownership for content is assigned **according to team or SIG responsibilities**. For example, Team Horizon "owns" articles related to [Developer Platform Overview](https://github.com/giantswarm/docs/blob/main/src/content/platform-overview/_index.md) as you can see the front matter.
+
+- [SIG Docs]({{< relref "/docs/organizational-structure/sigs/docs/_index.md" >}}) is responsible for overall content coherence and consistency.
+
+- [SIG User Experience]({{< relref "/docs/organizational-structure/sigs/ux/_index.md" >}}) (UX) takes responsibility for the overall UI and user experience, having the docs site online and available, have a user-friendly search etc.
+
+## Editing content
+
+In general, modifying content works as follows:
+
+The [giantswarm/docs](https://github.com/giantswarm/docs) repository is where all the documentation content lives. To modify content, clone that repository, create a branch, provide a pull request (PR).
+
+**Note:** You can [run the docs site locally](https://handbook.giantswarm.io/docs/product/docs/docs-dev/) to preview your changes.
```

Comment:
Dead link. And shouldn't we rather link to `docs/README.md`?

Reply:
yeah this will be available soon, needs to be migrated